nemo/collections/nlp/modules/common/huggingface/camembert.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 The Google AI Language Team Authors and
+# The HuggingFace Inc. team.
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Could you change this line to `# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.`